tools/c7n_mailer/c7n_mailer/record.py

```diff
@@ -1,150 +0,0 @@
-# Copyright 2016 Capital One Services, LLC
```

Comment:
This whole file was thrown into utils, there seem to be various random functions in here, might as well condense it into utils.